testing/build.gradle.kts

```diff
@@ -78,3 +82,7 @@ publishing {
 signing {
     sign(publishing.publications.getByName("language-server-testing"))
 }
+
+ktlint {
+    version = "1.2.1"
+}
```

Comment:
Another option would be to extract these to the root gradle project

Reply:
This would require making the root project a kotlin project, we might think about it in the future.